content/docs/guides/webassembly/resources.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Additional Resources"
+weight: 4
+description: |
+  Additional Resources for using TinyGo with WASM/WASI
+---
+
+Here are a few resources for helping learn more about using TinyGo with WebAssembly.
+
+**Wasm By Example**
+https://wasmbyexample.dev/
+
+**Are We Wasm Yet ? - Part 1**
+https://elewis.dev/are-we-wasm-yet-part-1
+
+**Are We Wasm Yet ? - Part 2**
+https://elewis.dev/are-we-wasm-yet-part-2
+
+**Shrink Your TinyGo WebAssembly Modules by 60%**
+https://www.fermyon.com/blog/optimizing-tinygo-wasm
```

Comment:
This one still uses `-wasm-abi=generic` which has been removed a while ago. Otherwise it seems fine.

Reply:
cc @technosophos